CodeBeam.MudBlazor.Extensions/Components/SignaturePad/MudSignaturePad.razor

```diff
@@ -0,0 +1,58 @@
+﻿@using Microsoft.JSInterop
+@using MudBlazor.Charts
+@using MudBlazor.Utilities
+@using System.Text
+@namespace MudExtensions
+@inject IJSRuntime JsRuntime
+@inherits ComponentBase
+
+<MudPaper Elevation="@Elevation" Class="@OuterClass">
+    <div style="@CanvasContainerStyle" class="@CanvasContainerClass">
+        <canvas @ref="_reference" id="@_id"></canvas>
+    </div>
+    <MudToolBar Style="@ToolbarStyle" Class="my-2">
+        <MudStack Row="true" Class="justify-content-between align-center">
+            @if (ShowLineWidth)
+            {
+                <MudNumericField T="decimal" Value="Options.LineWidth" ValueChanged="LineWidthUpdated" Label="@LocalizedStrings.LineWidth" Min="1" Step="1"></MudNumericField>
```

Comment:
Its better to add `Variant` `Dense` etc. parameters in the SignaturePad component and assign there like `Variant="@Variant"` so users can change variant with one parameter.